bootstrap.sh

```diff
@@ -15,6 +15,8 @@ kubectl apply -f .infrastructure/app/nodeport.yml
 kubectl apply -f .infrastructure/app/hpa.yml
 kubectl apply -f .infrastructure/app/deployment.yml
 
+kubectl apply -f .infrastructure/security/rbac.yml
```

Comment:
Ensure that the `rbac.yml` file in the `.infrastructure/security/` directory contains the necessary `ServiceAccount`, `Role`, and `RoleBinding` configurations as previously mentioned. This is crucial for the security setup.